### PR TITLE
Remove star imports

### PR DESCRIPTION
**Pull request by mattwthompson**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -24,10 +24,10 @@ per-file-ignores =
     openff/toolkit/topology/__init__.py:F401
     openff/toolkit/topology/molecule.py:
     openff/toolkit/topology/topology.py:F821
-    openff/toolkit/typing/chemistry/__init__.py:F401,F403
-    openff/toolkit/typing/engines/smirnoff/__init__.py:F401,F403
+    openff/toolkit/typing/chemistry/__init__.py:F401
```
Comment (j-wags) [inline setup.cfg:27]: (not blocking) This seems pretty brittle - Are there easy options to have this file not exist?

**Review by j-wags — APPROVED** [review]
LGTM!

File: setup.cfg (hunk 1 above)
Comment (mattwthompson) [reply setup.cfg:27]: The easiest route would be modifying the code to pass all `flake8` checks - something I'd be happy to do in a separate PR but not something that would be a trivial amount of work. I'd argue most of the checks `flake8` attempts to enforce are good ones.

**Comment by mattwthompson** [comment]
I had to resolve a merge conflict related to #1429; I believe it should be fine given that we're still maintaining the the import order of


```python3
from openff.toolkit.utils.utils import ...
from openff.toolkit.utils.toolkits import ...

**Pull request merged**